package.json

```diff
@@ -5,47 +5,58 @@
   "keywords": [
     "ember-addon"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rwjblue/ember-router-helpers.git"
+  },
   "license": "MIT",
   "author": "Robert Jackson <me@rwjblue.com>",
   "directories": {
     "test": "tests"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/rwjblue/ember-router-helpers.git"
-  },
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
-    "test": "ember try:each"
+    "lint:js": "eslint .",
+    "start": "yarn ember serve",
+    "test": "yarn ember test",
+    "test:all": "ember try:each",
+    "ember": "./node_modules/.bin/ember",
```

Comment:
This is superfluous with `yarn` (you can run `yarn ember` without any changes to scripts).

Reply:
this is mindblowing. thank you. lol